src/main/java/no/bibsys/lambda/deploy/handlers/SwaggerHubInfo.java

```diff
@@ -0,0 +1,56 @@
+package no.bibsys.lambda.deploy.handlers;
+
+import java.io.IOException;
+import no.bibsys.secrets.SecretsReader;
+import no.bibsys.utils.Environment;
+
+public class SwaggerHubInfo {
+
+
+    private final transient String apiId;
+    private final transient String apiVersion;
+    private final transient String swaggerOrganization;
+
+
+    public SwaggerHubInfo(Environment environment) {
+        this.apiId = environment.readEnvOpt("API_ID").orElse("simple-api");
+        this.apiVersion = environment.readEnvOpt("API_VERSION").orElse("1.0");
+        this.swaggerOrganization = environment.readEnvOpt("SWAGGER_ORG").orElse("axthosarouris");
```

Comment:
Hm, hardcoded values are not good. Maybe something we could provide in environment?